src/components/common/Timer/Timer.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from "react";
+import { useRef } from "react";
+// import { GameContext } from "../../contexts/context";
+import "./Timer.css";
+
+export const Timer = ({ timeForAWord, gameOver, gameStatus }) => {
+  let [timeToBeShown, setTimeToBeShown] = useState(`${timeForAWord}`);
+  const pathRef = useRef();
+  const [circleDasharray, setCircleDasharray] = useState(`283`);
+  // const [ms, setMs] = useState(0);
+  useEffect(() => {
+    startTimer();
+    // return () => clearInterval(ms);
+    // eslint-disable-next-line
+  }, []);
+
+  const TIME_LIMIT = timeForAWord;
+  const FULL_DASH_ARRAY = 283;
+  let timePassed = 0;
+  let timeLeft = TIME_LIMIT;
+  let timerInterval = null;
+  let ms = 0;
```

Comment:
don't use abbreviations as variable names. It is not always the case that the reviewer of the code is familiar with their meanings.
A better name, in this case, could be `timeInMilliSecs`

Reply:
Ok. got it